web/src/features/panels/zone/ZoneDetails.tsx

```diff
@@ -30,7 +32,7 @@ export default function ZoneDetails(): JSX.Element {
   if (!zoneId || Array.isArray(data)) {
     return <Navigate to="/" replace />;
   }
-
+  setZoneId(zoneId);
```

Comment:
This should probably live inside a useEffect and not just in root of the component - otherwise it will fire whenever component rerenders :)

Reply:
:)